html/semantics/scripting-1/the-script-element/moving-between-documents/resources/moving-between-documents-helper.js

```diff
@@ -48,7 +47,7 @@ window.didExecute = undefined;
 //
 // This helper is for tests where [1]/[2]/[3] are different.
 
-// In the spec, scripts are not executed only if [1]/[2]/[3] are all the same
+// In the spec, scripts are executed only if [1]/[2]/[3] are all the same
 // (or [1] is null and [2]==[3]).
 //
 // A check for [1]==[2] is in #prepare-a-script and
```

Comment:
It sounds like, if the Chromium/spec change lands, we can remove "but these are under debate" (or change it into "these used to be under debate; see link for background"). We can do that in either this PR or in a followup, depending on how the timing works out.